src/Hamiltonians/correlation_functions.jl

```diff
@@ -236,3 +236,153 @@ function get_offdiagonal(
     gd = exp(-im*g.d*Δp*2π/M)*gamma
     return new_add, ComplexF64(gd/M)
 end
+
+"""
+    SuperfluidCorrelator(d::Int) <: AbstractHamiltonian{Float64}
+
+Two-body operator for superfluid correlation between sites separated by `d` with `0 ≤ d < m`
```

Comment:
```suggestion
Operator for extracting superfluid correlation between sites separated by a distance `d` with `0 ≤ d < M`:
```
It is not a two-body operator but a one-body operator. But this is not relevant here at all.